docs/schema_registry_tutorial.rst

```diff
@@ -0,0 +1,532 @@
+   {"id":1}
+
+
+Schema Evolution and Compatibility
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Changing Schemas
+^^^^^^^^^^^^^^^^
+
+So far in this tutorial, you have seen the benefit of |sr-long| as being centralized schema management that enables client applications to register and retrieve globally unique schema ids.
+The main value of |sr|, however, is in enabling schema evolution.
+Similar to how APIs evolve and need to be compatible for all applications that rely on old and new versions of the API, schemas also evolve and likewise need to be compatible for all applications that rely on old and new versions of the schema.
+This schema evolution is a natural behavior of how applications and data develop over time.
+
+|sr-long| allows for schema evolution and provides compatibility checks to ensure that the contract between producers and consumers is not broken.
+This is especially important in Kafka because producers and consumers are decoupled applications that are sometimes developed by different teams.
+Compatibility checks on schemas allow producers and consumers to update independently and evolve their schemas independently, with assurances that they can read new and legacy data.
+|sr| can check compatibility of a new schema against just the latest registered schema, or if configured as transitive then it checks against all previously registered schemas, not just the latest one.
+
+These are the types of `compatibility types <https://docs.confluent.io/current/schema-registry/docs/config.html#avro-compatibility-level>`_:
+
+* ``FORWARD``: consumers using the latest registered schema can read data written by producers using the new schema
+* ``FORWARD_TRANSITIVE``: consumers using all previousely registered schemas can read data written by producers using the new schema
+* ``BACKWARD``: consumers using the new schema can read data written by producers using the latest registered schema
+* ``BACKWARD_TRANSITIVE``: consumers using the new schema can read data written by producers using all previously registered schemas
+* ``FULL``: the new schema is forward and backward compatible with the latest registered schema
+* ``FULL_TRANSITIVE``: the new schema is forward and backward compatible with all previously registered schemas
+* ``NONE``: schema compatibility checks are disabled
+
+By default, |sr| is configured for ``BACKWARD`` compatibility.
+You can change this globally or per subject, but for the remainder of this tutorial, leave the default compatibility level to `backward`.
+
+
+Failing Compatibility Checks
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+|sr| checks compatibility as schemas evolve to keep the producer-consumer contract.
```

Comment:
nit `contract [in takt].` ?

Reply:
typo: `intact`

Reply:
I changed this to:

`|sr| checks compatibility as schemas evolve to uphold the producer-consumer contract.`